src/sentry/search/events/builder/discover.py

```diff
@@ -881,6 +884,8 @@ def resolve_orderby(self, orderby: list[str] | str | None) -> list[OrderBy]:
                     validated.append(OrderBy(selected_column, direction))
                     break
 
+        if not validate:
+            return []
```

Comment:
If we're going to return an empty list, perhaps we can do this if block at the top of the function.